data/json/effect_on_condition.json

```diff
@@ -806,6 +806,14 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_discover_zomborg",
+    "recurrence": 1,
+    "global": true,
+    "condition": { "compare_int": [ { "global_val": "monsters_nearby", "id": "mon_zomborg" }, ">", { "const": 0 } ] },
```

Comment:
You can try this :
`"condition": { "math": [ "u_monsters_nearby('mon_zomborg')", ">", "0" ] },`.
However a recurrence of 1 is putting more stress on CPU. Please consider deactivating this eoc once you found a zomborg.

Reply:
You could also make an acheivement that requires you to kill a zomborg and then tie off that for zero impact.

Reply:
> You can try this : `"condition": { "math": [ "u_monsters_nearby('mon_zomborg')", ">", "0" ] },`. 
That didn't work for me, said it was an unrecognized function. However, making it a `u_val` fixes the issue.

Reply:
Weird how GitHub has no problem with it, though.